inference-engine/tests_deprecated/unit/engines/vpu/get_vpu_scale_from_ir_tests.cpp

```diff
@@ -7,111 +7,122 @@
-//             .setOutDepth(1).setInputPort(Port({1, 1, 1, 1}));
-
-//     layerId = netBuilder.addLayer({{layerId}, {weightsId}}, convBuilder);
-//     netBuilder.addLayer({PortInfo(layerId)}, Builder::OutputLayer("output"));
-
-//     auto network = Builder::convertToICNNNetwork(netBuilder.build());
-
-//     CNNLayerPtr layer;
-//     network->getLayerByName("Convolution", layer, nullptr);
-
-//     auto model = frontEnd->buildInitialModel(*network);
-//     auto middleEnd = passManager->buildMiddleEnd();
-
-//     auto checkWeightWasChanged = [this, network, layer](const float scale) {
-//         layer->params["vpu_scale"] = toString(scale);
-//         auto model = frontEnd->buildInitialModel(*network);
-//         auto middleEnd = passManager->buildMiddleEnd();
-//         middleEnd->run(model);
-//         for (const auto& stage : model->getStages()) {
-//             if (stage->name() == "Convolution") {
-//                 auto content = stage->input(1)->content()->get<ie_fp16>();
-//                 EXPECT_EQ(scale, PrecisionUtils::f16tof32(content[0]));
-//             }
-//         }
-//     };
-
-//     checkWeightWasChanged(32);
-//     checkWeightWasChanged(64);
-
-//     const auto maxVal = std::numeric_limits<float>::infinity();
-//     layer->params["vpu_scale"] = toString(maxVal);
-//     model = frontEnd->buildInitialModel(*network);
-//     middleEnd = passManager->buildMiddleEnd();
-//     middleEnd->run(model);
-
-//     for (const auto& stage : model->getStages()) {
-//         if (stage->name() == "Convolution") {
-//             EXPECT_EQ(stage->attrs().get<float>("scaleFactor"), maxVal);
-//         }
-//     }
-// }
+TEST_F(VPU_AddVpuScaleTest, CanAddVpuScaleToNetwork) {
+    InitCompileEnv();
+
+    auto& env = CompileEnv::get();
+    CompilationConfig config{};
+    config.irWithVpuScalesDir = "/";
+    env.updateConfig(config);
+
+    std::shared_ptr<ngraph::Function> function;
+
+    {
+        auto input = std::make_shared<ngraph::opset1::Parameter>(ngraph::element::f16, ngraph::Shape{4, 2, 2});
+        input->set_friendly_name("input");
+        auto weights = ngraph::opset1::Constant::create(ngraph::element::f16, ngraph::Shape{2, 2}, {1});
+        auto bias = ngraph::opset1::Constant::create(ngraph::element::f16, ngraph::Shape{2}, {1});
+        auto fc = std::make_shared<ngraph::op::FullyConnected>(input, weights, bias, ngraph::Shape{4, 2, 2});
+        fc->set_friendly_name("FullyConnected");
+        auto result = std::make_shared<ngraph::op::Result>(fc);
+        ngraph::ResultVector results { result };
+        ngraph::ParameterVector params {input };
+        function = std::make_shared<ngraph::Function>(results, params);
+    }
+
+    auto network = InferenceEngine::CNNNetwork(function);
+    auto model = frontEnd->buildInitialModel(network);
+
+    const auto getFullyConnectedStage = [&]() -> Handle<StageNode> {
```

Comment:
I think this would be more compact and easier to get:
```
const auto& stages = model->getStages();
const auto stageIt = std::find_if(stages.begin(), stages.end(), [&model](const Stage& stage) {
    return stage->type() == StageType::FullyConnected;
});
ASSERT_NE(stageIt, stages.end());
return *stageIt;
```